client/reader/post-options/_style.scss

```diff
@@ -54,7 +54,24 @@
 	}
 }
 
+.popover__menu {
+
+	.gridicon {
```

Comment:
This selector is not specific enough to the reader and is being picked up by other popover menus, like in /pages. I suggest adding a component class name to the reader popover menu.

<img width="291" alt="screen shot 2016-05-13 at 4 15 34 pm" src="https://cloud.githubusercontent.com/assets/1270189/15264378/737480ce-1927-11e6-995b-f454988217d4.png">